resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterManager.scala

```diff
@@ -90,7 +89,7 @@ private[spark] class KubernetesClusterManager extends ExternalClusterManager wit
     new KubernetesClusterSchedulerBackend(
         sc.taskScheduler.asInstanceOf[TaskSchedulerImpl],
         sc,
-        bootStrap,
+        initBootStrap,
```

Comment:
Should the input into `KubernetesClusterSchedulerBackend` include multiple `Option(BootstrapSteps)` or should this be re-architectured into some form of a step based structure as it had been done in the Driver